pydoctor/model.py

```diff
@@ -373,6 +375,18 @@ def report(self, descr: str, section: str = 'parsing', lineno_offset: int = 0) -
             f'{self.description}:{linenumber}: {descr}',
             thresh=-1)
 
+    @property
+    def docstringlinker(self) -> 'epydoc2stan.DocstringLinker':
+        """
+        Returns an instance of L{epydoc2stan.DocstringLinker} suitable for resolving names
+        in the context of the object scope. 
+        """
+        if self._linker is not None:
+            return self._linker
+        from pydoctor.epydoc2stan import _CachedEpydocLinker
+        self._linker = _CachedEpydocLinker(self)
+        return self._linker
```

Comment:
The best would be to avoid this workaround cyclic imports. 
In order to do that, we should move all code related to the concrete docstring linkers in a new module: `docstringlinker.py`. I wonder if this change should happen in this PR though ?

Reply:
Better for a separate PR, but also add all this info in a code comment :)